src/site/filters/liquid.js

```diff
@@ -237,28 +237,23 @@ module.exports = function registerFilters() {
     item ? item.sort((a, b) => a.entityId - b.entityId) : undefined;
 
   liquid.filters.eventSorter = item => {
-    const sorted = item.sort((a, b) => {
-      const aTime = Math.floor(
-        new Date(a.fieldDate.startDate).getTime() / 1000,
-      );
-      const bTime = Math.floor(
-        new Date(b.fieldDate.startDate).getTime() / 1000,
-      );
-      // Sort order needs to be oldest first.
-      const sorter = aTime - bTime;
-      return sorter;
-    });
+    const sorted =
+      item !== null
+        ? item.sort((a, b) => {
+            const aTime = Math.floor(
+              new Date(a.fieldDate.startDate).getTime() / 1000,
+            );
+            const bTime = Math.floor(
+              new Date(b.fieldDate.startDate).getTime() / 1000,
+            );
+            // Sort order needs to be oldest first.
+            const sorter = aTime - bTime;
+            return sorter;
+          })
+        : '';
```

Comment:
This is confusing block for me. I think it could be reduced to -

```js
const moment = require('moment');

// .... other code

const sorted = item && item.sort((a, b) => {
   const start1 = moment(a.fieldDate.startDate);
   const start2 = moment(b.fieldDate.startDate);
   return start1.isAfter(start2);
});
```

Reply:
:+1: @ncksllvn